redaxo/src/addons/backup/pages/export.php

```diff
@@ -43,6 +43,9 @@
     // ------------------------------ FUNC EXPORT
 
     $exportfilename = strtolower($exportfilename);
+    /**
+     * @psalm-taint-escape shell
+     */
```

Comment:
```suggestion
    /** @psalm-taint-escape shell */
```

Sowas würde ich einzeilig notieren.